internal/gengapic/client_init.go

```diff
@@ -254,7 +261,13 @@ func (g *generator) clientInit(serv *descriptor.ServiceDescriptorProto, servName
 		p("//")
 		g.comment(g.comments[serv])
 		p("func New%[1]sClient(ctx context.Context, opts ...option.ClientOption) (*%[1]sClient, error) {", servName)
-		p("  connPool, err := gtransport.DialPool(ctx, append(default%sClientOptions(), opts...)...)", servName)
+		p("  defaultOpts := default%sClientOptions()", servName)
```

Comment:
s/defaultOpts/opts/

(they get modified later, `defaultOpts = append(defaultOpts, ...)` is a bit confusing)

Reply:
`opts` is the identifier for the user-defined `[]ClientOption` parameter, which is why I used `defaultOpts`. Any other name ideas? `options`? `clientOpts`?

Reply:
oh, right. uhhh yeah, `mergedOpts` maybe? `clientOpts` seems good too.

Reply:
I'm gonna go with `clientOpts`. Thanks!